src/graph.jl

```diff
@@ -39,6 +39,35 @@ function min_bounding_rect(coords::Vector{Vector{Vector{Vector{Float64}}}})::Tup
 end
 
 
+function create_rtree(minimum_bounding_rects::Array{Tuple{Array{Float64,1},Array{Float64,1}},1})::LibSpatialIndex.RTree
+    """ Given an array of minimum bounding rectangles, constructs an R-Tree
+        that will make it easier to identify candidates for intersecting nodes.
+    """
+    rtree = LibSpatialIndex.RTree(2)
+    # insert an MBR for each polygon in the RTree
+    for (i, mbr) in enumerate(minimum_bounding_rects)
+        LibSpatialIndex.insert!(rtree, i, mbr[1], mbr[2])
```

Comment:
`mbr[1]` is the coordinate of the lower left corner, `mbr[2]` is the coordinate of the upper right corner

Reply:
this should be a comment in the code

Reply:
done!